app/helpers/patients_helper.rb

```diff
@@ -1,2 +1,17 @@
 module PatientsHelper
+
+  def bootstrap_class_for flash_type
+    { success: 'alert-success', error: 'alert-danger', warning: 'alert-warning'}[flash_type.to_sym]
```

Comment:
It initially was just a string but that was not picking it up after which I switched it over `.to_sym` and it worked.